apps/code-examples/src/app/code-examples/forms/input-box/inline-help/demo.component.html

```diff
@@ -13,6 +13,7 @@ <h2>New member form</h2>
           </label>
           <sky-help-inline
             class="sky-control-help"
+            ariaLabel="Show help info for First name"
```

Comment:
```suggestion
            ariaLabel="Information about First name"
```
Unless there was some reason that the First name and Last name fields are different than the other fields with help inline buttons, we should stick to the same wording that we use elsewhere.